modules/gapi/cmake/DownloadOneVPL.cmake.in

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(oneVPL-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(oneVPL
+  GIT_REPOSITORY    https://github.com/oneapi-src/oneVPL.git
+  GIT_TAG           origin
+  CMAKE_ARGS        -DCMAKE_INSTALL_PREFIX=${ONEVPL_PROJECT_PREFIX}
+  SOURCE_DIR        "${ONEVPL_PROJECT_PREFIX}/src"
+  BINARY_DIR        "${ONEVPL_PROJECT_PREFIX}/build"
+  INSTALL_DIR       ""
```

Comment:
use clone & build & install during opencv configure step in case of WITH_ONEVPL is ON **and** VPL package is not found in predefined places **and** VPL is not  sourced